packages/components/tree-v2/src/tree-node.vue

```diff
@@ -62,57 +60,42 @@ import {
   treeNodeEmits,
   treeNodeProps,
 } from './virtual-tree'
+import type { CheckboxValueType } from '@element-plus/components/checkbox/src/checkbox'
```

Comment:
Better to insert a blank line between `import ...` and `import type ...`. 
**Same in other places**

```suggestion
import type { CheckboxValueType } from '@element-plus/components/checkbox'
```